orchestra/todos/api.py

```diff
@@ -83,7 +83,7 @@ def _add_template_todo(
 
         todo = Todo(
             task=task,
-            description=template_todo['description'],
+            title=template_todo['description'],
```

Comment:
the JSON schema I commented above has this be description -> title. if you don't want to have to do the data migration, you can also undo the JSON schema update for now.